code/analytics/build.gradle

```diff
@@ -67,10 +67,8 @@ dependencies {
     testImplementation fileTree(include: ['*.aar'], dir: 'libs')
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test:runner:1.5.2'
-    androidTestImplementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
-    androidTestImplementation('com.adobe.marketing.mobile:identity:2.0.0-SNAPSHOT'){
-        transitive = false
-    }
+    androidTestImplementation 'com.adobe.marketing.mobile:core:2.0.0'
+    androidTestImplementation 'com.adobe.marketing.mobile:identity:2.0.0'
```

Comment:
Use "2.+" here for the test dependency.

Reply:
@yangyansong-adbe for test app/tests we will always want to use with latest available source code and catch issues early. I am thinking we can use 2.+ for now for these test targets and we logged a task to look into ways to update to latest available version and lock in the version similar with the ios podlock

Reply:
I saw that we changed to use 2.+ in another PR, so I will not change it here in this PR. Please check and make the correct change in the newer PR. 
cc @kevinlind @emdobrin